checks/http.go

```diff
@@ -174,7 +174,7 @@ func (c *HTTPChecker) Check(ctx *context.Context, extConfig external.Check) pkg.
 
 	connection, connectionData, err := ctx.GetConnectionTemplate(check.Connection)
 	if err != nil {
-		return results.Invalidf("error getting connection  %v", err)
+		return results.Failf("error getting connection  %v", err)
```

Comment:
This should be invalid ?

Reply:
hmm I guess could be both.

You could run into db timeout or kubernetes API call timeout issue.